### PR TITLE
Covid

### PR DESCRIPTION
**Pull request by AminaBuhari**
Hi Reviewer 😃, 
The covid-19-tracker is a mobile web application to check a list of covid metric update from different countries. 
An API that provides numeric data was used to build the webapp around it. The webapp have several pages:

-  One page with a list of items that could be filtered by the country. 
- One page for the item details. 
- One page for selecting the date you need update.

To highlight how the website was built :- 
[Narrativa API](https://covid19tracking.narrativa.com/index_en.html): COVID-19 data was intergrated in bulding the app. 
Unit tests for pure functions was created in my Redux code.
React and redux was used to build the app. 

Happy Reviewing

### CODE REVIEW

File: src/Components/Data.js [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+        </p>
+
+      </div>
+    </button>
+  );
+};
+
+Data.propTypes = {
+  date: PropTypes.number.isRequired,
```
Comment (phelian23) [inline src/Components/Data.js]: - [ ] The data type of the date prop supplied is `string`, but your app requires `number`. Kindly change it.

**Review by phelian23 — CHANGES_REQUESTED** [review]
Hi @AminaBuhari, 

Good job so far!
There are some issues that you still need to work on to prepare your project for the final evaluation but you are almost there!

## Required changes
- [ ] Kindly record a video presenting the features of the project as requested by the project requirements.

## Suggested changes
_Check the comments under the review._

You can use as many of my suggestions as you want. If there is anything you would like to skip - feel free to do that. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- x
- y
- z

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: src/Components/Data.js (hunk 1 above)
Comment (AminaBuhari) [reply src/Components/Data.js]: Great will make the change. Thank, you.

**Review by julie-ify — CHANGES_REQUESTED** [review]
Hi @AminaBuhari, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Required Changes ♻️
_Check the comments under the review._

- [ ] **[REQUIRED]** As requested by the previous code reviewer, kindly record a video presenting the features of the project as follows;

- The maximum length of the video is 5 minutes.
- The video contains a clear and objective description of the project in professional English. 
- The topic and purpose of the project are made clear.
- The video contains a demo of the project, highlighting all the project features.
- The video contains a highlight about a piece of the code the student finds interesting or is proud of making.

- [ ] **[OPTIONAL]** I suggest you fix your Netlify deployment checks so that your PR looks more professional and to avoid any errors the failed checks might cause.


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by AminaBuhari** [comment]
@julie-ify i did leave a video

**Comment by julie-ify** [comment]
@AminaBuhari, sorry but I didn't see any video presentation link in your PR or Readme. Please where is it located so that next reviewer can easily find it?

**Comment by AminaBuhari** [comment]
https://drive.google.com/file/d/1JJOX25vCWxYoIAykwViHWed0v1-_RjCO/view?usp=sharing

LINK TO VDEO

**Comment by AminaBuhari** [comment]
> @AminaBuhari, sorry but I didn't see any video presentation link in your PR or Readme. Please where is it located so that next reviewer can easily find it?

i added it to the additional information section on the dashboard while submitting the PR

**Comment by KDlamini** [comment]
> https://drive.google.com/file/d/1JJOX25vCWxYoIAykwViHWed0v1-_RjCO/view?usp=sharing
> 
> LINK TO VDEO

Hi @AminaBuhari,
Next time please either include the link to your video on the PR description or the README file. 👍🏾

**Review by KDlamini — APPROVED** [review]
### Hi @AminaBuhari 🙋‍♀️,  

Great work!
Your feature is complete! There is nothing else to say other than... it's time to merge it. :shipit: 
Congratulations!!!

# Status: Approved ✔️ 
 
Cheers 🥂  and Happy coding!!! 👯 
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @KDlamini  in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**